lib/plugins/pushover.js

```diff
@@ -14,8 +14,29 @@ function init (env) {
 
   var pushoverAPI = setupPushover(env);
 
-  pushover.send = function wrapSend(notify, callback) {
-    var selectedKeys = notify.isAnnouncement ? pushoverAPI.announcementKeys : pushoverAPI.userKeys;
+  function selectKeys (notify) {
+    var keys = null;
+
+    if (notify.isAnnouncement) {
+      keys = pushoverAPI.announcementKeys;
+    } else if (levels.isAlarm(notify.level)) {
+      keys = pushoverAPI.alarmKeys;
+    } else {
+      keys = pushoverAPI.userKeys;
+    }
+
+    return keys;
+  }
+
+  pushover.send = function wrapSend (notify, callback) {
+    var selectedKeys = selectKeys(notify);
+
+    if (selectedKeys.length == 0) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/940842039/issues/source?bid=2452379&fileBranchId=2847143#l34)